source/uwp/Renderer/dll/CppWinRTIncludes.h

```diff
@@ -54,6 +54,7 @@ namespace winrt
     // In order to avoid "namespace not defined" errors we have to define the namespace here too.
     namespace AdaptiveCards::Rendering::Uwp{}
     using namespace ::winrt::AdaptiveCards::Rendering::Uwp;
+    namespace render_xaml = ::winrt::AdaptiveCards::Rendering::Uwp;
```

Comment:
I see that in most of the places you simply use `winrt::` namespace to refer to renderer types and you've only declared `render_xaml` to use in cases where types exist in both objectModel and renderer namespaces?
like `enum ActionMode` for example?:) 

just want to make sure that is the intent:)